### PR TITLE
Extend README.md template

### PR DESCRIPTION
**Pull request by wiegandm**
This commit provides some example for possible README content, along
with slight rewording and the suggestion to put larger installation and
development instructions into separate files.

### CODE REVIEW

File: README-template.md [hunk 1]
```diff
@@ -62,13 +66,26 @@ This project is maintained by [Greenbone Networks GmbH](https://www.greenbone.ne
...
+
+    pip install --user pipenv
+
+Afterwards run
+
+    pipenv install --dev
+
+in the checkout directory of X (the directory containing the Pipfile) to install all dependencies including the packages only required for development.
+
+If there are more specific suggestions for development or suggestions only relevant to specific environments or use cases, consider creating a separate DEVELOPMENT.md file and [linking it here](DEVELOPMENT.md).
```
Comment (davidak) [inline README-template.md]: A common file for that is CONTRIBUTING.md. It is even displayed to new contributors on GitHub when creating a PR.

https://help.github.com/articles/setting-guidelines-for-repository-contributors/

File: README-template.md [hunk 2]
```diff
@@ -62,13 +66,26 @@ This project is maintained by [Greenbone Networks GmbH](https://www.greenbone.ne
 
 ## Contributing
 
-Your contributions are highly appreciated. Please [create a pull request](https://github.com/greenbone/X/pulls) on GitHub. For bigger changes, please discuss it first in the [issues](https://github.com/greenbone/X/issues).
+Your contributions are highly appreciated. Please [create a pull request](https://github.com/greenbone/X/pulls) on GitHub. For bigger changes, please start a discussion with the development team via the [issues section at github](https://github.com/greenbone/X/issues) first.
```
Comment (davidak) [inline README-template.md]: Starting a discussion is not enough. A consent has to be found. So "discuss it with the maintainers" would describe the process more precisely.

**Comment by davidak** [comment]
@bjoernricks i think it's too complex to set up when adding 2 files would also solve it. A CONTRIBUTING.md is even shown to new contributors when creating a PR. When Install and Usage section get's too big, an additional documentation makes sense.

**Comment by bjoernricks** [comment]
Setting up sphinx and readthedocs is really easy. Maybe we create another template ;-)

File: README-template.md (hunk 1 above)
Comment (wiegandm) [reply README-template.md]: I was thinking more along the lines of providing suggestions for the development process than for contributing the results of this process.

But I think this is too specific for a template, I rephrased the sentence to be more generic and less focused on a particular file or format.

File: README-template.md (hunk 2 above)
Comment (wiegandm) [reply README-template.md]: Agreed, I have rephrased the sentence.

**Pull request merged**